zenodo/modules/spam/views.py

```diff
@@ -126,13 +127,33 @@ def delete(user_id):
         return render_template('zenodo_spam/delete.html', **ctx)
 
 
-def _evaluate_user_domain(email):
-    email_domain = email.rsplit('@', 1)[-1].lower()
+def normalize_email(email):
+    email = email.lower()
+    username, domain = email.rsplit("@", 1)
+    no_dots_username = username.replace(".", "")
+    domain = email[email.index("@") :]
```

Comment:
minor: why not reuse the `domain` variable from the split above?

Reply:
Thank you for the suggestion :)